.github/workflows/notify-pr-close.yml

```diff
@@ -0,0 +1,24 @@
+name: Notify on PR close
```

Comment:
although this is cool :p I'm not too convinced of its usefulness, for a few reasons:
 1. the comment will go into a PR that may, or may NOT, have been opened by someone with write access. So in some cases, the comment just adds noise (new email) instead of adding value
 2. it works as a reminder for PRs, but what about direct pushes to `main`? I know the saying is "it is better than nothing", but it feels like a half-implementation 
 3. it isn't always relevant, i.e. we won't always want to release, so why send a reminder?

If you're really keen on keeping this and giving it a try, I won't oppose to it, but my gut feeling is that it will become a nuisance that we all just ignore cause we (writers) know what the process is to get a new release.

Reply:
Ah okay, good points. I will just ditch it for now.

Reply:
Removed in https://github.com/canonical/rocks-toolbox/pull/2/commits/b284546e3b81a8032bd2d5e99bcc671ae914b6de.